lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
```

Comment:
If we're going to be keeping this around we might want to change the name to something like "BasicLayout" or "SimpleLayout" to make it a bit clearer what the differences between the layout engines are.